testData/inspections/struct-initialization/literalValueWithoutCompositeLit.go

```diff
@@ -0,0 +1,13 @@
+package foo
+
+type S struct {
+	t int
+}
+
+type B struct{
+	S
+}
+
+func main() {
+	var  _  = []B{ S: {<weak_warning descr="Unnamed field initializations">1<caret></weak_warning>}}
```

Comment:
invalid test:
```
[]B {
 // S is invalid key because in array literals we use integer keys (indicies) 
    S : {  // expected literal type is B (parent literal has []B type)
           1 // expected key is `S` (type is B)
     }, 
}
```